### PR TITLE
Storybook visual tests

### PR DESCRIPTION
**Pull request by PippoRaimondi**
## Description of change

The intent of this PR is introduce storybook visual test setup into our pipeline

## Test instructions

steps:
- make start-storybook
- make visual-component-tests

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"

- [ ] Has the branch been rebased to master?
- [ ] Automated tests (Any of the following when applicable: Unit, Functional or Acceptance)
- [ ] Manual compatibility testing (Browsers: Chrome, Firefox, IE11, Safari)

### CODE REVIEW

**Review by ambrozic — APPROVED** [review]

**Review by rachaelcodes — APPROVED** [review]

File: test/visual-component/.eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "rules": {
+    "no-undef": "off"
+  }
+}
```
Comment (mforner13) [inline test/visual-component/.eslintrc.json:5]: Missing final line

**Comment by paulgain** [comment]
I think the Docker (the latest) docs need updating: https://github.com/uktrade/data-hub-frontend/blob/b23d8f0f6a7c6438be32ac6b4ba21c25af5de9c8/docs/Docker.md

**Review by mforner13 — COMMENTED** [review]
I wonder if we could have a separate make command for starting storybook without rebuilding, as it would be quite a wait every time? Although I realise we could just run `docker-compose -p dh -f docker-compose.storybook.yml up`

**Comment by paulgain** [comment]
There's lots of errors (just after Webpack hits 100%) bringing up the containers:
error

**Comment by PippoRaimondi** [comment]
> I wonder if we could have a separate make command for starting storybook without rebuilding, as it would be quite a wait every time? Although I realise we could just run `docker-compose -p dh -f docker-compose.storybook.yml up`

Yes, that's not a bad idea... there is always the option to start it locally as well `npm run storybook` ... I could add in another PR the option to start without rebuilding, not only for storybook...

**Comment by paulgain** [comment]
> I wonder if we could have a separate make command for starting storybook without rebuilding, as it would be quite a wait every time? Although I realise we could just run `docker-compose -p dh -f docker-compose.storybook.yml up`

I agree, it takes a long time to being these containers up despite having the images there.

**Comment by PippoRaimondi** [comment]
> **There are lots of errors (just after Webpack hits 100%) bringing up the containers:**
> 
> error

That error is related to the broken storybook in master, which has been fixed as part of this PR... make sure you have the latest of the branch

**Review by mforner13 — APPROVED** [review]

**Comment by paulgain** [comment]
> > **There are lots of errors (just after Webpack hits 100%) bringing up the containers:**
> > error
> 
> That error is related to the broken storybook in master, which has been fixed as part of this PR... make sure you have the latest of the branch

Yep, it now works as expected.

**Comment by PippoRaimondi** [comment]
> I think the Docker (the latest) docs need updating: https://github.com/uktrade/data-hub-frontend/blob/b23d8f0f6a7c6438be32ac6b4ba21c25af5de9c8/docs/Docker.md
> 
> The docs page you've updated is out of date.

agreed, will do that in another card if that's ok... this was captured and mentioned by Rachel as well...

**Pull request merged**